cli/azd/cmd/cmd_help.go

```diff
@@ -236,6 +236,10 @@ func getCommandsDetails(cmd *cobra.Command) (result string) {
 	max := 0
 	var lines []string
 	for _, childCommand := range childrenCommands {
+		if !childCommand.IsAvailableCommand() {
```

Comment:
Fixes hidden commands from showing up. Run `azd auth` in the current released version, you'll see that `token` shows up.